src/main.ts

```diff
@@ -11,6 +11,7 @@ import * as options from './options';
 
     const environment = options.getEnvironment();
     const sourcemaps = options.getSourcemaps();
+    const dist = options.getDist();
```

Comment:
this is where the env var `INPUT_DIST` is used, although it doesn't look like it.